engine/src/main/java/io/camunda/zeebe/engine/state/appliers/EventAppliers.java

```diff
@@ -62,6 +64,11 @@ public EventAppliers(final MutableZeebeState state) {
     registerProcessMessageSubscriptionEventAppliers(state);
     registerTimeEventAppliers(state);
     registerProcessEventAppliers(state);
+
+    register(DecisionIntent.CREATED, new DecisionCreatedApplier(state.getDmnDecisionState()));
+    register(
+        DecisionRequirementsIntent.CREATED,
+        new DecisionRequirementsApplier(state.getDmnDecisionState()));
```

Comment:
🔧 The DecisionRequirementsApplier should reflect the intent in its name as well as it only applies the state for the CREATED event.